src/common/error/errors/missing-data-error.ts

```diff
@@ -2,6 +2,7 @@ import StandardError from './standard-error';
 
 export enum MissingDataErrorType {
     MissingBillingAddress,
+    MissingShippingAddress,
```

Comment:
I don´t see that we are using this exception anywhere, do we still need it?

Reply:
No, I forgot remove it, I'll do that 👍